career/general_personality.md

```diff
@@ -0,0 +1,93 @@
+## Characteristics of a Programmer
+
+Here's some sample questions that you might encounter when an potential employer want to assess your characteristics that can signal if you make a good addition to their programming team.
+
+**If the interviewer does not ask questions about these topics, you as a candidate should try to bring them up yourself.**
+
+### Describe your experience with different programming languages
+
+Technical skills
+
+One mistake many hiring managers make is hiring from a checklist of requirements. Instead of requiring three years of C++ and one year of Java, look at the big picture. A programmer who has worked mostly in an older language but has recently added a new language to his certifications may be the perfect fit, since his years of experience give him a great foundation in other areas of programming.
+
+### What do you do to keep your programming skills current?
+
+Willingness to learn
+
+Technology is always evolving and the skills and abilities a programmer has today will likely be outdated in a few years. It’s important to find a programmer who has an interest in following the latest trends and is eager to participate in any continuing education opportunities that are available.
+
+### How do you handle bugs in your code?
+
+Debugging
+
+Creating code is only part of a programmer’s job. When software doesn’t work as expected, a programmer is expected to get to the root of the problem quickly and effectively. Instead of spending hours blindly making changes, search for a programmer who prefers to carefully investigate his code and research possible issues until an answer is found.
+
+### Describe your ideal work environment
+
+Work environment match
+
+Some programmers require complete silence to concentrate, while others thrive in chaos. A worker’s personal preferences are an important part of his productivity, so it’s best to be clear about your office environment up front to avoid problems after the person is hired.
+
+### What are your passion projects/hobbies?
+
+Passion for the work
+
+This is a tricky one. Many employers think that true programmers must be “computer geeks”, spending their time gaming, building servers or creating apps for friends. While this passion isn't a necessity, it's often a way to find top-shelf candidates. People that attend our bootcamp are not code obsessives with no other dimension to their personality. They do not devote every single moment of their life to coding and the don't live the stereotyped developer lifestyle. But we still argue that they are passionate - in a way that makes them, in a long run, more valuable to an organisation than people who code every second of every day.
```

Comment:
`the don't live the stereotyped developer`  they not the

Reply:
Done 👍